### DISCUSSION TITLE
Cotalker puede obtener coordenadas de una dirección ingresada por campos de textos

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#395] [2. Questions and Answers] [answered]
Hola

Link de referencia: https://github.com/Cotalker/documentation/discussions/192
Quisiera saber lo siguiente:

En un formulario: si tuviera 4 campos: 1.- Dirección, 2.- Número, 3.- Comuna, Región, y los ingresara: podría obtener la latitud y longitud de la zona que indiqué?.

Lo que quiero es que en el momento que se crea una task, el asset guarde los campos de latitud y longitud de lo indicado. La componente GPS me sirve, siempre y cuando creo la task en el lugar mismo, pero si los ingreso desde otro lugar no me permitiría registrar la zona en donde estuve.

Por otro lado, se puede dejar dinámico el componente GPS, en donde me permite mover en el mapa y marcar la posición deseada?

Si no quedó claro me avisan para explicarlo de mejor manera

Gracias =>

**Comment by acedward** [comment, marked as answer]
Respondiendo una cosa a la vez

1) Si ingresas los cuatro campos
Dirección Número Region Comuna; se puede convertir a su latitud y longitude por servicios externos como google-maps.
Habría que escribir un FAAS para realizar este cálculo

2) Lo del mapa dinámico, está en el roadmap futuro - pero no está con fecha de implementación aún.

**Reply by NicolasPLG** [reply]
hola @acedward muchas gracias por tu respuesta y entendí muy bien el punto 1. Tendría que pedir ayuda en la elaboración del faas.

Gracias =>

**Reply by z0rro2k** [reply]
hola, pudiste solucionarlo con faas? podrias compartir info acerca de faas? gracias

**Reply by NicolasPLG** [reply]
Hola @z0rro2k ocupé los campos lat y lon del componente gps y los agregué en la coleccion de mi campo

```
,"latitude":"$VALUE#data|[find=>identifier=location_land_opportunity]|process|0","longitude":"$VALUE#data|[find=>identifier=location_land_opportunity]|process|1"}}
```

Te recomiendo levantar otra discusión nueva dejando referencia ésta, ya que esta discusión está "resuelta"

=>
